common/util_test.go

```diff
@@ -74,3 +98,17 @@ func TestFlushRoutine(t *testing.T) {
 		i++
 	}
 }
+
+func TestGetProjectName(t *testing.T) {
```

Comment:
I suppose this probably isn't needed anymore, right? 😅

Reply:
yup! removed